### PR TITLE
feat: init docusaurus website

### PR DESCRIPTION
**Pull request by ankithans**

### CODE REVIEW

File: docs/doc1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (wtrocki) [inline docs/doc1.md:1]: Obviously further work will be required to get this done.
I would recommend you to clean not needed files (like this one and then we can push it to upstream branch so many people can collaborate.

File: website/blog/2016-03-11-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (wtrocki) [inline website/blog/2016-03-11-blog-post.md:1]: Remove

File: website/blog/2017-09-25-testing-rss.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (wtrocki) [inline website/blog/2017-09-25-testing-rss.md:1]: Remove

File: website/i18n/en.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (wtrocki) [inline website/i18n/en.json:1]: Remove

File: website/sidebars.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+  "docs": {
+    "Docusaurus": ["doc1"],
```
Comment (wtrocki) [inline website/sidebars.json:3]: Provide proper categories and link existing docs.

File: website/siteConfig.js [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    caption: 'User1',
+    // You will need to prepend the image path with your baseUrl
+    // if it is not '/', like: '/test-site/img/image.jpg'.
+    image: '/img/undraw_open_source.svg',
+    infoLink: 'https://www.facebook.com',
+    pinned: true,
+  },
+];
+
+const siteConfig = {
```
Comment (wtrocki) [inline website/siteConfig.js:23]: Provide reliable values for those. Use github pages as link.

File: website/static/css/custom.css [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+@media only screen and (max-width: 1023px) {
+}
+
+@media only screen and (min-width: 1400px) {
+}
+
+@media only screen and (min-width: 1500px) {
+}
```
Comment (wtrocki) [inline website/static/css/custom.css:23]: All resources bellow should be removed.

**Comment by wtrocki** [comment]
Thanks for your contribution. Let's remove not neded/placeholder files and we will see if we can bring this work upstream.

**Comment by ankithans** [comment]
- removed unwanted files
- provide categories to existing docs
- fill ```siteConfig.js``` with reliable info

**Comment by wtrocki** [comment]
Can you put some info on front page and transform it to work with link.

File: .dockerignore [hunk 8]
```diff
@@ -0,0 +1,2 @@
+*/node_modules
```
Comment (wtrocki) [inline .dockerignore:1]: Remove

File: Dockerfile [hunk 9]
```diff
@@ -0,0 +1,10 @@
+FROM node:lts
```
Comment (wtrocki) [inline Dockerfile:1]: Remove

File: docker-compose.yml [hunk 10]
```diff
@@ -0,0 +1,18 @@
+version: "3"
```
Comment (wtrocki) [inline docker-compose.yml:1]: Remove

File: website/blog/2017-04-10-blog-post-two.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (wtrocki) [inline website/blog/2017-04-10-blog-post-two.md:1]: Remove

File: website/core/Footer.js [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+                height="58"
+              />
+            )}
+          </a>
+          <div>
+            <h5>Docs</h5>
+            <a href={this.docUrl('doc1.html')}>
+              Getting Started (or other categories)
+            </a>
+            <a href={this.docUrl('doc2.html')}>Guides (or other categories)</a>
```
Comment (wtrocki) [inline website/core/Footer.js:37]: Let's just remove those for now

File: website/core/Footer.js [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+            <a href={this.docUrl('doc3.html')}>
+              API Reference (or other categories)
+            </a>
+          </div>
+          <div>
+            <h5>Community</h5>
```
Comment (wtrocki) [inline website/core/Footer.js:43]: Add community chat and remove everything else

File: website/core/Footer.js [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+                  data-share="true"
+                  data-width="225"
+                  data-show-faces="false"
+                />
+              </div>
+            )}
+          </div>
+        </section>
+
+        <a
```
Comment (wtrocki) [inline website/core/Footer.js:98]: Removd

File: website/pages/en/index.js [hunk 15]
```diff
@@ -0,0 +1,73 @@
+const React = require("react");
+
+function HomeSplash(props) {
```
Comment (wtrocki) [inline website/pages/en/index.js:3]: Might be good to put something here like logo at least and some general info
See features in readme

File: website/siteConfig.js [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = [
```
Comment (wtrocki) [inline website/siteConfig.js:12]: Is this needed

File: website/yarn.lock [hunk 17]
```diff
@@ -0,0 +1,6622 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (wtrocki) [inline website/yarn.lock:2]: ReMob

File: website/siteConfig.js (hunk 16 above)
Comment (ankithans) [reply website/siteConfig.js:12]: no! removing

File: website/pages/en/index.js (hunk 15 above)
Comment (wtrocki) [reply website/pages/en/index.js:3]: No design needed but some info will be cool

**Comment by ankithans** [comment]
- just realized, I used docusaurus v1. So updated it to v2.
- add features on home

**Comment by wtrocki** [comment]
@ankithans amazing work. last part would be to enable GitHub pages on your fork and share URL. Will look to merge it tommorow

**Comment by ankithans** [comment]
- https://ankithans.github.io/graphql-link/

**Comment by wtrocki** [comment]
Amazing contribution! Really neat work.
Still some small issues but it will be faster for me to fix them rather than comment things. 

@ankithans you nailed the job! Thank you so much for help on this!

**Pull request merged**